api_server/src/request/boot_source.rs

```diff
@@ -29,13 +29,15 @@ mod tests {
     use super::*;
 
     #[test]
-    fn test_into_parsed_request() {
+    fn test_into_parsed_request_same() {
```

Comment:
I don't think we need two tests here. We can have just one where initrd_path has a value.

Reply:
Ok!